resources/[system]/scoreboard/server.lua

```diff
@@ -0,0 +1,101 @@
+RegisterNetEvent("scoreboard:getPlayers")
+AddEventHandler("scoreboard:getPlayers", function()
+  TriggerClientEvent("scoreboard:receivePlayers", source, players)
+end)
+
+columns = {
+  {
+    friendlyName = "Player ID",
+    defaultValue = 0,
+    position = 0
+  },
+  {
+    friendlyName = "Name",
+    defaultValue = "***Invalid***",
+    position = 1
+  }
+}
+
+players = {}
+
+RegisterNetEvent("scoreboard:requestColumns")
+AddEventHandler("scoreboard:requestColumns", function()
+  players[source] = {}
+  for id, columnData in pairs(columns) do
+    players[source][columnData.friendlyName] = columnData.defaultValue
+  end
+  players[source]["Player ID"] = source
+  players[source]["Name"] = GetPlayerName(source)
```

Comment:
.. in fact, why is this event adding the local player's state in the first place?